lib/linguist/heuristics.rb

```diff
@@ -160,6 +160,20 @@ def call(data)
       end
     end
 
+    disambiguate "M", "Mathematica", "Matlab", "Mercury", "Objective-C" do |data|
+      if /^[ \t]*@(interface|class|protocol|property|end|synchronised|selector|implementation)\b/.match(data)
```

Comment:
I think `^[ \t]*` can be simplified to `^\s*`.

http://rubular.com/r/s2usmgMO3A